src/game/main.cpp

```diff
@@ -253,7 +291,6 @@ void Create_Window()
     ShowWindow(app_hwnd, show_cmd);
     UpdateWindow(app_hwnd);
 
-    g_applicationHInstance = app_hinstance;
```

Comment:
Moving the original variables around will make it more difficult to compare function against original.